relayer/hyperspace/hyperspace.go

```diff
@@ -340,27 +338,27 @@ func (hyperspaceCommander) GeneratePath(srcChainID, dstChainID, pathName, homeDi
 // Hyperspace does not have paths, just two configs
 func (hyperspaceCommander) UpdatePath(pathName, homeDir string, filter ibc.ChannelFilter) []string {
 	panic("[UpdatePath] Do not call me")
-	
+
 }
 
 func (hyperspaceCommander) GetChannels(chainID, homeDir string) []string {
-	panic("Panic because hyperspace will panic")
-	/*fmt.Println("[hyperspace] Get Channels")
-	configFilePath := path.Join(homeDir, chainID + ".config")
+	//panic("Panic because hyperspace will panic")
```

Comment:
Can just remove the panic instead of commenting it

Reply:
Will do 👍